### ISSUE TITLE
Debugger shows nothing

### ISSUE THREAD

**Issue by angelovAlex** [shuky19/sublime_debugger#59]
Thanks for the great project, I hope I will be able to test and enjoy it.
Looks like I missed a step, but I can't figure out where to look for logs or something that can help find the issue.
So I installed ruby 2.1.0 and byebug and ruby debugger for sublimetext on macOS 10.12.3, (sublime build 3126), opened a ruby file in sublimetext and started the debugging of current file.
Many windows appeared at the bottom but they all empty, nothing in Output.
I also changed settings of the package to show debug logs and changed rvm to rbenv, but still nothing in windows.
The only message that I noticed was "Stopping...". It appears if I select "Tools" -> "Build" and then if I start debugging it immediately says "Stopping...".
So what I missed?

**Comment by iandol** [comment]
Yes, this tool is broken on macOS, it certainly doesn't work for me using Ruby 2.4.1 on 10.12.6 (after adding 2.4.1 to the supported list). Perhaps you can only use 2.0.0 in which case this debugger is not very useful for most users.
